### DISCUSSION TITLE
Imvdb down or taken offline

### DISCUSSION THREAD

**Discussion by jaycedk** [RandomNinjaAtk/arr-scripts#274] [General]
Hi all.
Do any of you ladies or gents, know if Imvdb is down or taken offline ?
I noticed the lidarr script is unable to connect to Imvdb.
Trying to access Imvdb.com its just plain blank.
Looking in browser console it displays Error 500 internal server error.

I know this is an upstream issue, and is by no means @RandomNinjaAtk fault.
I'm just curious.

**Comment by RandomNinjaAtk** [comment]
Might be time to remove/disable imvdb from the script.... Not sure what else can be done...

**Comment by jaycedk** [comment]
Let give it some more time.
It might resurface again, no need to rush.
Is there a alternativ for music videos ?

**Comment by jaycedk** [comment]
Nice It looks like they are online again 😁

**Reply by TheLazyGeekGuy** [reply]
And down since one week again .... :-/

**Reply by jaycedk** [reply]
And they are back.
